lib/install.ps1

```diff
@@ -533,29 +533,34 @@ function dl_urls($app, $version, $manifest, $bucket, $architecture, $dir, $use_c
 
         $extract_dir = $extract_dirs[$extracted]
         $extract_to = $extract_tos[$extracted]
-
+        $Installer = installer $manifest $architecture
```

Comment:
Why first uppercase character as name inside code?

Reply:
Whole codebase is lowercased for internal variables. Except Parameters, which follow powershell's Pascal Case.

Reply:
Refactored `dl_urls`.